Library/Homebrew/caveats.rb

```diff
@@ -136,11 +136,12 @@ def app_caveats
   def elisp_caveats
     return if f.keg_only?
     if keg && keg.elisp_installed?
-      <<-EOS.undent
+      <<-EOS.undent_________________________________________________________72
```

Comment:
Can we not just manually do this without adding the giant `_________________________________________________________72` line?